rest/src/main/java/io/syndesis/rest/v1/handler/integration/IntegrationHandler.java

```diff
@@ -108,6 +121,44 @@ public Integration get(String id) {
         return integration;
     }
 
+
+    @GET
+    @Path("/{id}/export.zip")
```

Comment:
as the name used here is also used for storing the file, I would chose a more expressive name like 'ignite-integration-export.zip' (or 'syndesis-integraiton-export.zip').

Or, alternatively and probably better, set the Http-Header for `Content-Type: application/octet-stream` and `Content-Disposition: attachment; filename="syndesis-integration-export.zip"` (with customization for fuse-ignite in production)

Reply:
It's not really used to control the file name.  The ui module is in control of the the file name.  see: https://github.com/syndesisio/syndesis-ui/blob/master/src/app/integrations/detail-page/detail.component.ts#L413